client/src/test/java/tds/session/SessionTest.java

```diff
@@ -19,4 +22,64 @@ public void shouldBeInstantiated() {
         assertThat(session.getType()).isEqualTo(0);
         assertThat(session.getStatus()).isEqualTo("closed");
     }
+
+    @Test
+    public void shouldBeOpen() {
```

Comment:
I am fine either way.  I usually add the 'it' in front of should out of habit, but I can learn new tricks.